CIScripts/TestIntegration.ps1

```diff
@@ -37,11 +41,13 @@ $DockerDriverConfiguration = [DockerDriverConfiguration] @{
     Username = $Env:DOCKER_DRIVER_USERNAME;
     Password = $Env:DOCKER_DRIVER_PASSWORD;
     AuthUrl = $Env:DOCKER_DRIVER_AUTH_URL;
-    ControllerIP = $Env:DOCKER_DRIVER_CONTROLLER_IP;
     NetworkConfiguration = $DockerNetworkConfiguration;
 }
 
 $TestConfiguration = [TestConfiguration] @{
+    ControllerIP = $Env:CONTROLLER_IP;
```

Comment:
You'll notice that I moved this field from `DockerDriverConfiguration` struct and renamed the field. Already added new fields to jenkins job params. I kept `DOCKER_DRIVER_CONTROLLER_IP` to not break anything but we'll need to remove it eventually.